src/lib.rs

```diff
@@ -1,4 +1,54 @@
 //! Winit is a cross-platform window creation and event loop management library.
+//! 
+//! # Installation
+//! 
+//! `winit` can be added to `Cargo.toml` as a dependency. It can be added via `cargo add`.
+//! 
+//! ```bash
+//! $ cargo add winit
+//! ```
+//! 
+//! To only enable the X11 backend on Free Unix[^unix] systems, disable default features
+//! and enable the `x11` feature.
+//! 
+//! ```bash
+//! $ cargo add winit --no-default-features --features x11
+//! ```
+//! 
+//! To only enable the Wayland backend on Free Unix systems, disable default features
+//! and enable the `wayland` feature.
+//! 
+//! ```bash
+//! $ cargo add winit --no-default-features --features wayland
+//! ```
+//! 
+//! These features have no effect on systems that are not Free Unix.
+//! 
+//! ## Dependencies
+//! 
+//! Dependencies on non-system libraries is managed through Cargo. For the X11
+//! backend, the following Ubuntu packages or their equivalents must be installed.
+//! 
+//! - `libx11-dev`
+//! - `libxcb1-dev`
+//! - `libxi-dev`
+//! - `libxcbcommon-dev`
+//! - `libxcbcommon-x11-dev`
+//! 
+//! For the Wayland backend, the following Ubuntu packages or their equivalents
+//! must be installed.
+//! 
+//! - `libwayland-dev`
+//! - `libxcbcommon-dev`
```

Comment:
@kchibisov Are there any additional systems dependencies needed on Wayland? I'm not sure about this.

Reply:
you'd need `libfontconfig/freetype` once you enable `sctk-adwaita` feature.

Reply:
Indicated as such